packages/codemirror-extension/src/index.ts

```diff
@@ -157,22 +162,133 @@ function activateEditorCommands(
     lineWiseCopyCut
   } = CodeMirrorEditor.defaultConfig;
 
+  let toggleComment = 'Ctrl-/';
+  let toggleCommentMac = 'Cmd-/';
   /**
    * Update the setting values.
    */
   async function updateSettings(
     settings: ISettingRegistry.ISettings
   ): Promise<void> {
     keyMap = (settings.get('keyMap').composite as string | null) || keyMap;
-
-    // Lazy loading of vim mode
     if (keyMap === 'vim') {
+      // This setup for vim in the notebook is derived from extension built by
```

Comment:
Could the content of this `if` statement be mode to a separated ts file that would host static methods for vim, but also for upcoming keymaps. This would allow keeping this index.ts more readable and would allow keypmap implementer to have a place to focus on.

Reply:
Sounds good - will do